Bio/Phylo/NeXMLIO.py

```diff
@@ -138,7 +138,7 @@ def parse(self, values_are_confidence=False, rooted=False):
                 node_children = {}
                 root = None
 
-                child_tags = node.getchildren()
+                child_tags = list(node)
                 nodes = []
                 edges = []
                 for child in child_tags:
```

Comment:
`child_tags` is no longer used. Why not iterate just `node`?
```suggestion
                for child in node:
```